guide/src/wasm-bindgen-test/continuous-integration.md

```diff
@@ -53,3 +53,24 @@ test_script:
   - set GECKODRIVER=C:\Tools\WebDriver\geckodriver.exe
   - cargo test --target wasm32-unknown-unknown
 ```
+
+## GitHub Actions
+
+```yaml
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: jetli/wasm-pack-action@v0.2.0
+
+      - name: Cargo test
+        uses: actions-rs/cargo@v1.0.1
+        with:
+          command: test
+
+      - name: Wasm test
+        run: wasm-pack test --headless --chrome
```

Comment:
Can this test in both firefox and chrome, like the other examples?

Reply:
Sure.
Do you prefer that to be one step

```yaml
- name: Wasm test
  run: |
    wasm-pack test --headless --chrome
    wasm-pack test --headless --firefox
```

or two?

```yaml
- run: wasm-pack test --headless --chrome
- run: wasm-pack test --headless --firefox
```

Reply:
I guess two steps will make it easier to tell what fails 😄

Reply:
I figured the `-name` part is unnecessary here.
The default name becomes Run + the command
(i.e. Run wasm-pack test --headless --chrome)
which I think reads fine.